lib/utils/unicode-categories.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2018 mdsf project authors. Use of this source code is
```

Comment:
You should also mark this file as autogenerated for GitHub, to do this, create a `.gitattributes` file in the root of the repository and add the next line to it:
```git
/lib/utils/unicode-categories.js linguist-generated=true
```